src/test/java/io/jenkins/plugins/forensics/miner/RelativeCountTrendChartTest.java

```diff
@@ -0,0 +1,83 @@
+package io.jenkins.plugins.forensics.miner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.echarts.Build;
+import edu.hm.hafner.echarts.BuildResult;
+import edu.hm.hafner.echarts.ChartModelConfiguration;
+import edu.hm.hafner.echarts.LinesChartModel;
+import edu.hm.hafner.echarts.SeriesBuilder;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests RelativeCountTrendChart.
+ *
+ * @author Nikolas Paripovic
+ *
+ */
+class RelativeCountTrendChartTest {
+
+    private final RelativeCountTrendChart relativeCountTrendChart = new RelativeCountTrendChart();
+
+    @Test
+    void shouldCreate() {
+
+        Iterable<BuildResult<CommitStatisticsBuildAction>> buildResult = createBuildResults(false);
```

Comment:
Inline that call to make it readable:
```suggestion
        Iterable<BuildResult<CommitStatisticsBuildAction>> buildResult = new ArrayList<>();
```